tests/installation/live_installation.pm

```diff
@@ -48,7 +49,8 @@ sub run {
         assert_and_click 'live-upgrade';
     }
     else {
-        x11_start_program('xdg-su -c "/usr/sbin/start-install.sh"', target_match => 'maximize');
+        # On aarch64 there is sporadic issue when "Install" icon is clicked too long, so that context menu is called instead of opening the wizard.
+        is_aarch64 ? x11_start_program('xdg-su -c "/usr/sbin/start-install.sh"', target_match => 'maximize') : assert_and_click 'live-installation';
```

Comment:
Don't you mind to rewrite it with if condition?
I think it's too long for ternary operator, but that's matter of taste.
Other than that LGTM

Reply:
Changed.

Reply:
We need else branch, otherwise on arm we will start wizard and then try to click the icon =)